packages/data-point/lib/reducer-types/reducer-entity/factory.js

```diff
@@ -14,7 +12,6 @@ module.exports.type = REDUCER_ENTITY
  * @property {string} entityType - type of entity
  */
 function ReducerEntity () {
-  this[REDUCER_SYMBOL] = true
```

Comment:
@raingerber curious what is the reason of removing this property?

Reply:
it's not removed, it's just added from a different place:

https://github.com/ViacomInc/data-point/blob/freeze-reducers-in-main-factory/packages/data-point/lib/reducer-types/factory.js#L71

sorry that wasn't clear

Reply:
ohhh i c